pkg/satellite/stream/stream.go

```diff
@@ -277,6 +280,10 @@ func (ss *satelliteStream) openStream() error {
 
 	ss.conn = conn
 	ss.stream = stream
+	if ss.isVerbose && ss.streamId != "" {
```

Comment:
Is there ever a case where `streamId` would be empty?

Reply:
Yes, the first path here without -r option doesn't have streamId. The CLI set streamId when it receivs the very first data. This check is to prevent the CLI outputting streamId every time it receives new data.